packages/angular_devkit/core/src/workspace/json/writer.ts

```diff
@@ -175,7 +175,12 @@ function stringify(
 
     return content.replace(/\n/g, spacing);
   } else {
-    return JSON.stringify(value);
+    const content = JSON.stringify(value);
+    if (content.length > 60) {
+      return stringify(value, true, depth, indent);
```

Comment:
Can you add a comment explaining why this is necessary, and what the 60 is?

Reply:
removed.  It's not needed.